### PR TITLE
Add edge names

### PR DESCRIPTION
**Pull request by Marc-Week**
Added edge names to graph and on hover

### CODE REVIEW

File: lib/projects.js [hunk 1]
```diff
@@ -94,9 +94,29 @@ function getProjectRelationships (projectId, url) {
   }, error => Promise.reject(error));
 }
 
+/**
+ * Retrieve all the item types. Item types contain image urls which are then copied to each node so that they may be
```
Comment (RickyV33) [inline lib/projects.js]: This comment needs to change

File: test/lib/graph.js [hunk 2]
```diff
@@ -154,7 +154,6 @@ describe('Graph class module', () => {
       newGraph = new Graph(projectId, url);
       expect(newGraph.name).to.be.empty();
       expect(newGraph.nodes).to.be.empty();
-      expect(newGraph.edges).to.be.empty();
```
Comment (RickyV33) [inline test/lib/graph.js]: Why was this removed?

Comment (Marc-Week) [inline test/lib/graph.js]: Must of been a mistake

**Comment by chances** [comment]
LGTM, save for the merge conflicts

**Comment by MichaelLHansen** [comment]
LGTM
